frontend/components/navbar/Navbar.tsx

```
@@ -6,16 +6,24 @@ import SearchBar from "./SearchBar";
 
 export default function Navbar() {
   return (
-    <div className="bg-white fixed top-0 w-screen z-50">
-      <div className="flex items-center py-2 px-5 lg:px-8 xl:px-12">
-        <div className="mr-12">
-          <Logo />
+    <header className="bg-white fixed top-0 w-full z-50 box-border border-b-[1px] border-core-outline h-14">
+      <nav
+        className="flex items-center justify-between h-full px-5 lg:px-8 xl:px-12 max-w-full"
+        role="navigation"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)